osbs/build/build_requestv2.py

```diff
@@ -444,6 +449,17 @@ def render_resource_limits(self):
             resources['limits'] = limits
             self.template['spec']['resources'] = resources
 
+    def set_deadline(self, build_type):
+        if build_type == BUILD_TYPE_WORKER:
+            deadline_hours = self.user_params.worker_deadline.value
+        else:
+            deadline_hours = self.user_params.orchestrator_deadline.value
+
+        deadline_seconds = deadline_hours * 3600
```

Comment:
It would be great if we could disable the deadline entirely using config. For example, if `deadline_hours == 0` here.

Reply:
Fixed.